### ISSUE TITLE
RETURN OF OLD MONKEY TABLE

### ISSUE THREAD

**Issue by RSG250** [goonstation/goonstation#2331]
[BUG]

**Describe the bug**
While being a monkey, be wearing magboots and turn them on, proceed to flip self over table, after the bar completes you do not flip over table, however you can walk on top of it like the old monkey table movement

CURSED WIZARDRY

**To Reproduce**
1. Become as monkeh
2. wear magboots and turn them on
3. walk into table
3b. walk into table again.

**Expected behavior**
i honestly dont know, all i know is that this should NOT be the behavior it displays, i think the first time i did it i didnt realize my magboots were on so i expected normal monkey vaulting behavior, after realizing it was the magboots the second time i actually figured it out it was them

TLDR; not this, but something else maybe just not vaulting over the table when their on?
